src/java/com/unifina/signalpath/JavaCompilerErrorMessage.java

```diff
@@ -0,0 +1,21 @@
+package com.unifina.signalpath;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class JavaCompilerErrorMessage extends ModuleExceptionMessage {
+	public JavaCompilerErrorMessage(final int moduleIdHash, final Map<String, Object> msg) {
```

Comment:
Shouldn't pass around `Map`s anymore. For a solution, see comments in `ModuleExceptionMessage` and `AbstractJavaCodeWrapper`.